.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+name: CI
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clean
+        uses: easimon/maximize-build-space@v4
+        with:
+          root-reserve-mb: 8192
+          swap-size-mb: 1024
+          remove-dotnet: 'true'
```

Comment:
This action is amazing and gives us a ton more room for building. That avoids having to spin up our own infrastructure to build and publish this. The down side is that it can't be ran in a container, so the normal way of using `bilelmoussaoui/flatpak-github-actions` didn't work. Most of the code here is from that action though.